### PR TITLE
supervision

### PR DESCRIPTION
**Pull request by patriknw**

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -2,7 +2,7 @@ name := "akka-typed-blog"
 
 version := "1.0"
 
-scalaVersion := "2.11.11"
+scalaVersion := "2.12.2"
```
Comment (patriknw) [inline build.sbt]: Found a Java API oversight: https://github.com/akka/akka/issues/22961
I have intentionally used 2.11 exactly for this reason.

Comment (ktoso) [inline build.sbt]: Hah, good catch!

File: draft/supervision.md [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+description: ""
+author: Patrik Nordwall
+category: typed
+series_title: Introducing Akka Typed
+series_tag: typed
+tags: [actor,typed]
+---
+{% include JB/setup %}
+
+Supervision semantics in Akka Typed has changed slightly compared to supervision in classic untyped actors. Untyped actors are by default restarted when an exception is thrown. The default for typed actors is instead to stop the failing actor. In this blog post I will show how to install restarting supervision behaviors in Akka Typed.
```
Comment (ktoso) [inline draft/supervision.md]: semantics ... has -> have changed

File: draft/supervision.md [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+```scala
+val strategy = SupervisorStrategy.restart
+val worker = ctx.spawn(
+  Actor.restarter[RuntimeException](strategy).wrap(workerBehavior),
+  "worker")
+```
+
+That will restart the actor for all thrown `RuntimeException` or subclasses thereof.
+
+There are a few other strategies that you recognize from untyped actors, read the API documentation for details.
```
Comment (2m) [inline draft/supervision.md]: Link to the API would be nice here. Do we have it hosted somewhere?

Comment (patriknw) [inline draft/supervision.md]: yes, but that has to wait until we release from master

**Pull request merged**